documentation/ADRs/023_production_development.md

```diff
@@ -0,0 +1,43 @@
+## Production and Development Branches
+*Using production and development branches instead of main*
+
+| ADR Info            | Details           |
+|---------------------|-------------------|
+| Subject             | Production and Development Branches  |
+| ADR Number          | 023   |
+| Status              | proposed   |
+| Author              | Borbála   |
+| Date                | 29.10.2024.     |
+
+## Context
+Historically, the `main` branch was used for production and development work occurred directly via feature branched pulled from `main`. This required increased testing and restrictions on `main` such as making it protected or creating a GitHub action that prevents merging if branch is behind. These additional restrictions made the development cumbersome and increased the risk of merging unstable features.
+
+Therefore, we decided to establish a dedicated `production`(renamed from `main`) branch and a `development` branch, from where the feature branches are pulled instead of the former `main`. The `development` branch is synchronized with the `production` branch.
+
+
+## Decision
+
+### Overview
+
+We renamed the `main` branch to `production` and established a new `development` branch as the main source for feature branches. Development work will now occur in feature branches off `development`, which will be synced periodically with `production`. The exact synchronization procedure is To Be Discussed... The `production` branch remains protected with the condition that two reviewers should accept the Pull Request in order to be merged. There GitHub action that prevents merging if branch is behind still applies to `production` branch (although this workflow ensures that the `development` branch is never behind `production` branch) and a new action is created for the same reason targeting `development` branch.
```

Comment:
Can you add that "The exact synchronization procedure is be decided and described in ADR023". Then make a empty ADR for that (no need to fill it now) and paste the link here?